graph-tutorial/app.js

```diff
@@ -1,23 +1,20 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-const createError = require('http-errors');
-const express = require('express');
-const path = require('path');
-const cookieParser = require('cookie-parser');
-const logger = require('morgan');
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
```

Comment:
Use `const` instead of `var` for `require` statements (old tutorial files may have this wrong)